src/main/java/Duke/InvalidDateFormat.java

```diff
@@ -0,0 +1,7 @@
+package duke;
+
+public class InvalidDateFormat extends DukeException {
```

Comment:
Should this exception name follow the existing naming pattern you've established i.e. [description]Exception?

```suggestion
public class InvalidDateFormatException extends DukeException {
```

Reply:
Thanks for all the comments! Your suggestions make sense, and I'll be sure to improve with those in mind!